src/styles/sass/main.scss

```diff
@@ -6,87 +6,340 @@
 @import "type";
 
 // Navigation Control
-
-.controls {
-  color: $nypl-white;
+// Temp grayscaling for ease of styling...
+body {
+  // filter: grayscale(1);
+  // filter: grayscale(0);
   font-family: $sans-serif-stack;
   font-size: $basefont;
-  height: 4.65rem;
+  margin: 0 auto !important;
```

Comment:
just to be clear you're suggesting:
removing  the inline `style` on the `<body>` tag and change the scss to this:
`body {
  font-family: $sans-serif-stack;
  font-size: $basefont;
  margin: 0 auto;
  overflow: hidden;

  @media (min-width: 45rem) {
    width: 40rem;
  }
}`